pymess/backend/sms/__init__.py

```diff
@@ -1,46 +1,46 @@
 import logging
-
+from collections import OrderedDict
 from datetime import timedelta
 
 from django.utils import timezone
-from django.utils.translation import ugettext
+from django.utils.translation import ugettext_lazy as _
 
 from chamber.exceptions import PersistenceException
 
-from pymess.backend import BaseBackend, send_template as _send_template, send as _send
-from pymess.config import settings, get_sms_template_model, get_sms_sender
+from pymess.backend import BaseBackend, BaseController
+from pymess.backend import send as _send
+from pymess.backend import send_template as _send_template
+from pymess.config import get_sms_sender, get_sms_template_model, settings, get_sms_testing_sender
 from pymess.models import OutputSMSMessage
 from pymess.utils import fullname
 
-
 LOGGER = logging.getLogger(__name__)
 
 
-class SMSBackend(BaseBackend):
+class SMSController(BaseController):
     """
-    Base class for SMS backend containing implementation of concrete SMS service that
-    is used for sending SMS messages.
+    Controller class for SMS delegating message to correct SMS backend
     """
-
     model = OutputSMSMessage
 
+    def get_backends_dict(self):
+        return OrderedDict([
+            (get_sms_testing_sender(), settings.SMS_TESTING_PHONES),
```

Comment:
opět tohle se má nastavovat v settings. Cesty k backendům per čísla. Takto to není univerzální

Reply:
I have renamed the default sender backend config then to `_1`